azure-iot-device/azure/iot/device/iothub/pipeline/pipeline_stages_iothub_mqtt.py

```diff
@@ -97,100 +95,91 @@ def _execute_op(self, op):
                 )
             )
 
-            # make a callback that can call the user's callback after the reconnect is complete
             def on_reconnect_complete(reconnect_op, error):
```

Comment:
>on_reconnect_complete [](start = 16, length = 21)

should this have a complete ?

Reply:
No. A worker op automatically completes its parent op.

Also this callback no longer exists.

---
In reply to: [347018814](https://github.com/Azure/azure-iot-sdk-python/pull/392#discussion_r347018814) [](ancestors = 347018814)